### PR TITLE
Replace linearprogress with Progress from ODS

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

- Replace linearprogress with Progress from ODS
- Fix empty link breaking the proposal creation

Task1: [APP-2775](https://aragonassociation.atlassian.net/browse/APP-2775)
Task2: [APP-2800](https://aragonassociation.atlassian.net/browse/APP-2800)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2775]: https://aragonassociation.atlassian.net/browse/APP-2775?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2800]: https://aragonassociation.atlassian.net/browse/APP-2800?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/containers/configureCommunity/minParticipation.tsx [hunk 1]
```diff
@@ -163,9 +164,14 @@ export const MinParticipation: FC = () => {
...
                   <LinearProgressContainer>
-                    <LinearProgress
-                      max={govTokenTotalSupply}
-                      value={govTokenTotalSupply * (value / 100)}
+                    <Progress
+                      value={
+                        govTokenTotalSupply
+                          ? ((govTokenTotalSupply * (value / 100)) /
+                              govTokenTotalSupply) *
+                            100
```
Comment (jamesej) [inline src/containers/configureCommunity/minParticipation.tsx]: `((govTokenTotalSuppy * (value / 100)) / govTokenTotalSupply) * 100` is actually equal to just `value`

File: src/components/multisigMinimumApproval/minimumApproval.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ const MinimumApproval: React.FC<Props> = ({max, min, ...rest}) => {
 
         <ProgressWrapper>
           <LinearProgressContainer>
-            <LinearProgress max={max} value={value <= max ? value : max} />
+            <Progress value={((value <= max ? value : max) / max) * 100} />
```
Comment (jamesej) [inline src/components/multisigMinimumApproval/minimumApproval.tsx:33]: Original `LinearProgress` has a container div with the style `w-full` set on it. `Progress` doesn't have this and so does not extend to the full width of `LinearProgressContainer`, which looks wrong. This is likely a problem with the component itself.

**Review by jamesej — CHANGES_REQUESTED** [review]
The issue around the new `Progress` component not having `w-full` class probably affects all instances except in Voting Terminal.

File: src/containers/configureCommunity/minParticipation.tsx (hunk 1 above)
Comment (sepehr2github) [reply src/containers/configureCommunity/minParticipation.tsx]: > `((govTokenTotalSuppy * (value / 100)) / govTokenTotalSupply) * 100` is actually equal to just `value`

hahaha, nice catch

**Review by cgero-eth — COMMENTED** [review]
Something's off on the min-participation progress:
Screenshot 2024-02-14 at 15 16 04

**Review by cgero-eth — APPROVED** [review]
All good for me, thanks @sepehr2github!

**Pull request merged**